src/open_inwoner/utils/middleware.py

```diff
@@ -0,0 +1,109 @@
+import abc
+
+from django.conf import settings
+from django.shortcuts import redirect, resolve_url
+from django.urls import reverse
+from django.urls.exceptions import NoReverseMatch
+
+from furl import furl
+
+
+def get_always_pass_prefixes() -> tuple[str, ...]:
+    return (
+        settings.MEDIA_URL,
+        settings.PRIVATE_MEDIA_URL,
+        settings.STATIC_URL,
+        reverse("login"),
```

Comment:
I suspect that this will indeed be necessary, not being able to logout from the registrationscreen was a DigiD audit failure point 2 years ago

Reply:
`/digid/` is added as a prefix below so that logout should work now, but to also pass regular logout, `reverse("logout")` would have to be added separately

Reply:
Ahh that makes sense, lets leave it as is then